package.json

```diff
@@ -110,7 +112,7 @@
     "sri-update": "grunt build && node build/sri-update && git add sri-history.json",
     "sri-validate": "node build/sri-update --validate",
     "fmt": "prettier --write .",
-    "prepare": "husky",
+    "prepare": "husky && npm run unpatch && npm run patch",
```

Comment:
Do we actually need to unpatch first? I thought you had done that so we could get an unpatched version to test against. You patch-package just skips if you run it on an already patched package.

Reply:
You're correct 09a36d54